### ISSUE TITLE
Suggestion: per config nametag value

### ISSUE THREAD

**Issue by Yertled** [froobynooby/FarmControl#37]
This is a suggestion that would be incredebly beneficial to our server. I don't want FarmControl to either take ALL or NONE nametagged mobs, but only mobs I choose.
For example, I want FarmControl to kill named Villagers, but not named Zombies, if that makes sense?

From my understanding, it's not possible to choose which mob-type is killed with nametag or not, and that is a feature I would love to see.

**Comment by ZepsiZola** [comment]
I agree with this. There's a guy on my server who has a ton of nametagged zombies in one area to scare villagers. I want to disable the ai of those zombies but I can't because they're nametagged.

Pls let us configure per-profile whether we want to exemptify entities from a profile.
